presto-docs/src/main/sphinx/release/release-0.229.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.229
+=============
+
+General Changes
+_______________
+* Fix an issue that would cause query failure when calling :func:`geometry_to_bing_tiles` on certain degenerate geometries.
+* Add geospatial function :func:`line_interpolate_point`.
+* Add support for ``CREATE FUNCTION``.
+* Add support for using configuration property ``stage.max-tasks-per-stage`` to limit the number of tasks for scan.
+* Add support for passing ``X_Forwarded_For`` header from Proxy to coordinator.
+* Add configuration property ``experimental.internal-communication.max-task-update-size`` to limit the size of the ``TaskUpdate``.
+* Add configuration properties ``internal-communication.https.trust-store-path``, ``internal-communication.https.included-cipher``, and ``internal-communication.https.excluded-cipher``
+  to easily set common https configurations for all internal communications at one place.
+* Add peak task memory distribution of each stage to ``QueryStatistics``.
+
+Pinot Connector Changes
+_______________________
+* Add Pinot connector.
+
+SPI Changes
+___________
+* Add support for connectors to alter query plans involving ``UNION``, ``INTERSECT``, and ``EXCEPT``, by moving ``SetOperationNode`` to SPI.
+* Improve interface ``ConnectorPlanOptimizerProvider`` to allow connectors to participate in query optimization in two phases, ``LOGICAL`` and ``PHYSICAL``.
```

Comment:
Is this line a little too long? Maybe some others as well.

Reply:
I generally don't find this is an issue in the release notes as long as they're readable on Github and in IntelliJ. User will see generated HTML, which is irrelevant the line length here.